### PR TITLE
Update uniques for 3.19.0 Kalandra

### PR DESCRIPTION
**Pull request by Paliak**
Current changes only include 3.19 variants for unique helmets and belts. Will try adding more.

Overrides Wurm's molt level requirement to the level requirement as of 3.19.0 due to lack of support for variant level requirements

Unsure about the ordering on some mods.

### CODE REVIEW

File: src/Data/Uniques/helmet.lua [hunk 1]
```diff
@@ -1204,12 +1277,15 @@ Variant: Pre 1.0.0
...
+Variant: Pre 3.19.0
 Variant: Current
 Requires Level 17, 21 Dex, 21 Int
-(15-30)% increased Spell Damage
+{variant:1,2,3,5}(15-30)% increased Spell Damage
 +20 to Strength
-(20-30)% increased Lightning Damage
-+10% to Lightning Resistance
+{variant:1,2,3,5}(20-30)% increased Lightning Damage
+{variant:1,2,3,5}+10% to Lightning Resistance
```
Comment (Wires77) [inline src/Data/Uniques/helmet.lua:1286]:
```suggestion
{variant:1,2,3,4,5}(20-30)% increased Lightning Damage
{variant:1,2,3,4,5}+10% to Lightning Resistance
```

File: src/Data/Uniques/helmet.lua [hunk 2]
```diff
@@ -1204,12 +1277,15 @@ Variant: Pre 1.0.0
 Variant: Pre 2.0.0
 Variant: Pre 3.7.0
 Variant: Pre 3.17.0
+Variant: Pre 3.19.0
 Variant: Current
 Requires Level 17, 21 Dex, 21 Int
-(15-30)% increased Spell Damage
+{variant:1,2,3,5}(15-30)% increased Spell Damage
```
Comment (Wires77) [inline src/Data/Uniques/helmet.lua:1283]:
```suggestion
{variant:1,2,3,4,5}(15-30)% increased Spell Damage
```

**Review by Wires77 — CHANGES_REQUESTED** [review]
Some things need to be fixed on these uniques still.  Also `LevelReq:` should be used and should work if it's above the implicit section

File: src/Data/Uniques/belt.lua [hunk 3]
```diff
@@ -417,15 +417,17 @@ Immortal Flesh
...
 Variant: Current
 LevelReq: 50
 Implicits: 1
 {tags:life}+(25-40) to maximum Life
 {tags:life}+(75-100) to maximum Life
-{tags:life}Regenerate (67-75) Life per second
-{tags:mana}Regenerate (8-10) Mana per second
+{variant:1,2,3}{tags:life}(67-75) Life Regenerated per second
+{variant:4}{tags:life}(200-350) Life Regenerated per second
+{tags:mana}(8-10) Mana Regenerated per second
```
[the remark this answers is not in the document]
Comment (Paliak) [reply src/Data/Uniques/belt.lua:428]: Don't remember. Maybe it said that on the sheet ? Will change.

File: src/Data/Uniques/belt.lua [hunk 4]
```diff
@@ -444,13 +446,19 @@ Offering Skills have 50% reduced Duration
...
 Implicits: 1
 (20-30)% increased Stun Duration on Enemies
+{variant:2}{tags:physical}(25-40)% increased Physical Damage
 {tags:jewellery_attribute}+(40-50) to Strength
 {tags:physical}(25-40)% increased Global Physical Damage
 50% increased Flask Charges gained
 {variant:2}{tags:jewellery_resistance}+(20-25)% to all Elemental Resistances while you have at least 200 Strength
+{variant:3}{tags:jewellery_resistance}+(20-25)% to all Elemental Resistances
+{variant:3}10% chance to deal Double Damage while you have at least 200 Strength
+{variant:3}5% chance to deal Triple Damage while you have at least 400 Strength
```
[the remark this answers is not in the document]
Comment (Paliak) [reply src/Data/Uniques/belt.lua:461]: I guess wiki is wrong then. Will use trade site from now on. https://www.poewiki.net/wiki/The_Magnate

**Comment by Paliak** [comment]
I'm not 100% on the usage of LevelReq but i've made the requested changes.

**Review by Wires77 — APPROVED** [review]

**Pull request merged**